Cargo.toml

```diff
@@ -1,3 +1,5 @@
+cargo-features = [ "profile-rustflags" ]
```

Comment:
It's available only on `nighlty`, but we anyway compile iroha on `nighltly`.

Reply:
users don't necessarily need to compile on `nightly`, only building smart contracts seems to require `nightly`. Will this force us to always use `nightly` or just when using `profiling` profile?

Reply:
Build will fail on stable.

```
cargo +stable build 
error: failed to parse manifest at `/Users/erigara/Projects/iroha/Cargo.toml`

Caused by:
  the cargo feature `profile-rustflags` requires a nightly version of Cargo, but this is the `stable` channel
  See https://doc.rust-lang.org/book/appendix-07-nightly-rust.html for more information about Rust release channels.
  See https://doc.rust-lang.org/cargo/reference/unstable.html#profile-rustflags-option for more information about using this feature.
```

Reply:
Alternatively we could create `Makefile` or smt like this to pass all required flags.

Reply:
then I would remove it from `Cargo.toml` and pass it on the CLI

Reply:
unfortunately ni, so in such case we have to pass them trough `RUSTFLAGS` on build

Reply:
building Iroha on `nightly` must remain optional

Reply:
removed